pages/common/dvc-freeze.md

```diff
@@ -0,0 +1,13 @@
+# dvc freeze
+
+> Freeze stages in DVC pipeline.
+> This makes DVC to stop tracking changes in stage dependencies and re-execution untill unfreeze.
+> More information: <https://dvc.org/doc/command-reference/freeze>.
```

Comment:
Perhaps this will read better?
```suggestion
> Freeze stages in the DVC pipeline.
> This makes prevents DVC from tracking changes in stage dependencies and re-execution until unfreeze.
> See also `dvs unfreeze`.
> More information: <https://dvc.org/doc/command-reference/freeze>.
```